platform-api/index.md

```diff
@@ -0,0 +1,33 @@
+---
+
+ 
+
+copyright:
+
+  years: 2016
+
+ 
+
+---
+
+{:shortdesc: .shortdesc} 
+{:new_window: target="_blank"}
+
+# Platform API
+*Last updated: 9 May 2016*
+
+The Platform API allows programmatic access to Bluemix. Get a user's apps, launch server instances, create images, work with service instances and more.
+
+## Overview
+
+Bluemix is built using several open-source technologies, each with their respective API.
+
+- [Cloud Foundry API](https://apidocs.cloudfoundry.org)
+- [OpenWhisk API](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/openwhisk/openwhisk/master/core/controller/src/resources/whiskswagger.json)
+- [Containers API](http://ccsapi-doc.mybluemix.net/)
+- [OpenStack API](http://developer.openstack.org/api-ref.html) (virtual servers)
+
+
+## Authentication
+
+Use the [Basic Authentication Guide](basic-authentication-guide.html) to get an access token to include in the Authorization header of your request calls.
```

Comment:
@cpk1224 Similar to comment on the other file, lets make it clear this is not how you'd do it in production code.